### ISSUE TITLE
Find usages with IDEA 2016.3 EAP (build 163.5644.15), Lombok 1.16.10 and Lombok plugin 0.13.16 not working 

### ISSUE THREAD

**Issue by ThanksForAllTheFish** [mplushnikov/lombok-intellij-plugin#294]
Given this simple test class.

```
import lombok.Data;

@Getter //or @Data
public class TestClass {

    private String string;

    @Override
    public String toString() {
        getString();
        return string;
    }
}
```

code is generated (as shown) but find usages on the field retrieves no result. IDE event log is empty so don't know how to provide more insight.

For the record, switching the annotation (so, from `@Getter` to `@Data`) breaks code generation in the IDE (`maven` is still fine). Restarting the IDE invalidating the cache makes code generation working but not find usages (which is particularly annoying since then also refactoring features are broken, since they rely on usages).
Full project attached.

Thanks for any feedback
[test.zip](https://github.com/mplushnikov/lombok-intellij-plugin/files/515513/test.zip)

**Comment by Janmm14** [comment]
Maybe test with 2016.2.4?

**Comment by bric3** [comment]
Also it seems that _Analyse Data Flow_ don't find accessors to leading to missed usage

**Comment by ThanksForAllTheFish** [comment]
Sorry for the delay, I now tried version `2016.2.4` and no problem at all there, then I tried version `2016.2.5` and I was able to reproduce the problem described with the sample project attached.

I guess I will temporarily rollback to version `2016.2.4` as it is really annoying to miss refactoring features

**Comment by kreiger** [comment]
"Analyze data flow to here" still does not work with the current Lombok plugin version 0.15.17.2.

**Comment by mplushnikov** [comment]
closed as duplicate of #144 and #186
